### PR TITLE
Zooming with mouse wheel only possible with ctrl key

### LINKED ISSUE #461: Map: zoom avec ctrl + moulinette souris

### PR DESCRIPTION
**Pull request by kalbermattenm**
Fix #461 

@rufener, please review

### CODE REVIEW

File: front/src/components/MapHandler/MapHandler.vue [hunk 1]
```diff
@@ -91,7 +92,8 @@ export default {
 
         const interactions = new defaultInteractions({
           altShiftDragRotate: false,
-          pinchRotate: false
+          pinchRotate: false,
+          mouseWheelZoom:false
```
Comment (kalbermattenm) [inline front/src/components/MapHandler/MapHandler.vue:96]: Missed a space here, damned!

**Review by rufener — COMMENTED** [review]
Cool, fonctionne comme espéré. Merci

**Review by rufener — COMMENTED** [review]
Add some helper when mouse is on the map

**Review by rufener — APPROVED** [review]
Pour moi on peut merger

**Pull request merged**